packages/aws-cdk-lib/aws-ecs-patterns/lib/fargate/application-load-balanced-fargate-service.ts

```diff
@@ -152,4 +174,38 @@ export class ApplicationLoadBalancedFargateService extends ApplicationLoadBalanc
       throw new Error(`${name}: Must be a non-negative integer; received ${value}`);
     }
   }
+
+  /**
+   * Throws an error if containerCpu is greater than cpu or is a not positive integer.
+   * @param containerCpu The minimum number of CPU units to reserve for the container.
+   * @param cpu The number of cpu units used by the task. (default: 256)
+   */
+  private validateContainerCpu(containerCpu?: number, cpu: number = 256) {
+    if (containerCpu === undefined || Token.isUnresolved(containerCpu) || Token.isUnresolved(cpu)) {
+      return;
+    }
+    if (containerCpu > cpu) {
+      throw new Error('containerCpu must be less than to cpu');
```

Comment:
It would be user friendly to display the value specified.

```suggestion
      throw new Error(`containerCpu must be less than to cpu; received containerCpu: ${containerCpu}, cpu: ${cpu}`);
```

Reply:
https://github.com/aws/aws-cdk/pull/30920/commits/2b7a189843299d8a4386c906cd8edf5f5c779775